components/lib/utils/ObjectUtils.js

```diff
@@ -416,4 +416,31 @@ export default class ObjectUtils {
 
         return [];
     }
+
+    static mutateFieldData(data, field, value) {
+        // console.log(data,field,value)
```

Comment:
Rmeove console.log

Reply:
Sorry I should have checked before I pushed